Algorithm.Python/EmaCrossUniverseSelectionAlgorithm.py

```diff
@@ -32,76 +32,86 @@
 ### <meta name="tag" content="indicators" />
 ### <meta name="tag" content="universes" />
 ### <meta name="tag" content="coarse universes" />
-class EmaCrossUniverseSelectionAlgorithm(QCAlgorithm):
-
+class EmaCrossUniverse(QCAlgorithm):
```

Comment:
The main class and file name should share the same name: `EmaCrossUniverseSelectionAlgorithm`.